content/en/seachat/getting-started/02-quick-start.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Quickstart"
+description: "SeaChat Intro and getting started"
+lead: ""
+date: 2024-03-08 10:43:51.069 +0100
+lastmod: 2024-03-08 10:43:51.069 +0100
+weight: 202
+draft: false
+images: []
+aliases:
+  - /en/seachat/seachat-intro/02-quick-start/
+  - /seachat/seachat-intro/02-quick-start/
+---
+---
+* In just under 10 minutes, you'll have your own customized AI agent set up and ready to deploy on your website.
+* To get a quick taste of SeaChat, you can simply log into your SeaChat account now and follow our hands-on video tutorial to start.
+* For more detailed guidance, please consult the [SeaChat User Manual](/en/seachat/seachat-intro/02-how-to-create/). Should you encounter any issues or have questions, our [support team](#support) is on standby to assist you. Don't hesitate to reach out.
+
+---
+
+In this tutorial, we will create a SeaChat agent from scratch. This AI Agent will help us answer frequently asked questions about parking lots. We will walk you through the process of creating an AI agent, adding knowledge to its knowledge base, and testing and fine-tuning the agent to ensure it's ready for deployment.
```

Comment:
1. This AI agent will help answer FAQs about a parking lot business. 
2. don't say "deployment". The audience are no-code users. "... to ensure it's ready to answer all incoming questions from customers!